README.md

```diff
@@ -7,21 +7,24 @@
 
 ## Overview
 
-Magic Modules is a tool we use to autogenerate infrastructure-as-code tools for
-Google Cloud Platform. GCP ["resources"](https://cloud.google.com/docs/overview/#gcp_resources)
-are encoded in a shared data file, and that data is used to fill in
-"Mad Libs"-style templates across each of the tools Magic Modules generates.
+Magic Modules is a tool we use to autogenerate support in infrastructure-as-code
+tools for Google Cloud Platform. [GCP "resource"](https://cloud.google.com/docs/overview/#gcp_resources)
+definitions are encoded in a shared data file, and that data is used to fill in
+tool-specific "Mad Libs"-style templates across each of the tools Magic Modules
```

Comment:
```suggestion
tool-specific templates across each of the tools Magic Modules
```

Reply:
While I think using "mad libs" makes for a great talk at a conference, I think its a bit too campy here.